packages/components/src/Account/Auth/EmailSent.tsx

```diff
@@ -1,19 +1,35 @@
 import * as React from "react";
 import { RouteComponentProps } from "react-router-dom";
+import { CheckEmailSection, AuthPageFooterLink } from "./AuthStyledComponents";
+import { PageSubHeadingCentered, PageHeadingTextCentered } from "../../Heading";
 
 export interface AccountEmailSentProps extends Partial<RouteComponentProps> {
   isNewUser: boolean;
-  emailAddress?: string;
+  emailAddress: string;
+  onSendAgain?(): void;
 }
 
 export class AccountEmailSent extends React.Component<AccountEmailSentProps> {
   public render(): JSX.Element {
-    const emailAddress = this.props.emailAddress || this.props.location!.state.emailAddress;
+    const { emailAddress, onSendAgain } = this.props;
 
-    const { isNewUser } = this.props;
     return (
       <>
-        Check your email to {!isNewUser ? "sign in" : "register"}: <strong>{emailAddress}</strong>
+        <PageSubHeadingCentered>Check your email!</PageSubHeadingCentered>
```

Comment:
You can add this text component to the text file also

Reply:
Done.